schemas/file.yml

```
@@ -184,3 +184,21 @@
         https://www.iana.org/assignments/media-types/media-types.xhtml[IANA official types],
         where possible. When more than one type is applicable, the most specific
         type should be used.
+
+    - name: fork_name
+      level: extended
+      type: keyword
+      description: >
+        A fork is additional data associated with a filesystem object.
+
+        On Linux, a resource fork is used to store additional data with a filesystem object. A file always has at
+        least one fork, for the data portion and additional forks may exist.
+
+        On Windows, this is analogous to an Alternate Data Stream (ADS), and the default data stream for a file is just called $DATA.
```

```suggestion
        On Windows NTFS, this is analogous to an Alternate Data Stream (ADS), and the default data stream for a file is just called $DATA.
```

Should we explicitly call out NTFS?

---

I could go either way, we could just say "On Windows, ..."

---

Ok, I'm good either way as well. Author's choice then 😄